apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/sharing-modal/style.scss

```diff
@@ -0,0 +1,163 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.components-modal__screen-overlay {
```

Comment:
I think this ends up affecting all the modals after the the CSS has been loaded, not just this specific modal?

Reply:
Actually this isn't neccessary at all. Good catch, I added it to deal with the a8c "super admin bar" which puts the admin bar on top of everything, hiding the `X` button. But I realized this only applies for a12s, so I'll just remove this code.

Reply:
Maybe it should be changed in the a8c super admin bar plugin, I'll take a look at that 👀

Reply:
Oh interesting, it looks like it has just been addressed with the latest Gutenberg update, they now leave a gap at the top of the modal, which happens to fit the super admin bar perfectly 😄 

https://github.com/wordpress/gutenberg/pull/50039

Reply:
Great. :-) And modals can be dismissed with esc key anyway.